sqlalchemy_exasol/base.py

```diff
@@ -895,13 +744,7 @@ def fkey_rec():
             }
 
         fkeys = util.defaultdict(fkey_rec)
-        odbc_connection = self.getODBCConnection(connection)
-        if odbc_connection is not None and not self.use_sql_fallback(**kw):
-            constraints = self._get_foreign_keys_odbc(connection, odbc_connection, table_name=table_name,
-                                                      schema=schema_int, **kw)
-        else:
-            constraints = self._get_foreign_keys_sql(connection, table_name=table_name, schema=schema_int, **kw)
-        table_name = self.denormalize_name(table_name)
```

Comment:
hm, why did we have this here, is now the question, I can't see a reason, maybe a left over

Reply:
I don't know the IDE just pointed out it that it isn't used